executor/stale_txn_test.go

```diff
@@ -267,3 +267,16 @@ func (s *testStaleTxnSerialSuite) TestStalenessTransactionSchemaVer(c *C) {
 	// got an old infoSchema
 	c.Assert(schemaVer3, Equals, schemaVer1)
 }
+
+func (s *testStaleTxnSerialSuite) TestSetTransactionReadOnlyAsOf(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec(`SET TRANSACTION READ ONLY as of timestamp '2021-04-21 00:42:12'`)
```

Comment:
Should support expression? our bound read is a function call.

Reply:
It is already supported, the testcase updated.